hack/ci-e2e.sh

```diff
@@ -59,6 +59,108 @@ export BOOT_MAC_ADDRESS
 # These are the VBMC defaults (used since we did not specify anything else for `vbmc add`).
 export BMC_USER=admin
 export BMC_PASSWORD=password
+# This is the IP address of the host where the image server is running.
+IMAGE_FILE="cirros-0.5.2-x86_64-disk.img"
+export IMAGE_MD5SUM="b874c39491a2377b8490f5f1e89761a4"
+IMAGE_IP_ADDRESS=$(echo $BMC_ADDRESS | sed -n 's|ipmi://\([^:]*\).*|\1|p')
```

Comment:
I think we can make it even easier to understand. Let's create a new variable, e.g. `IP_ADDRESS` or `MINIKUBE_IP`. Then use this to set both the IMAGE_URL and the BMC_ADDRESS.

```
IP_ADDRESS="192.168.222.1"
BMC_ADDRESS="ipmi://${IP_ADDRESS}:${VBMC_PORT}"
IMAGE_URL="http://${IP_ADDRESS}/${IMAGE_FILE}"
```